src/lib.rs

```diff
@@ -83,12 +83,22 @@
 //! assert_eq!(3, *map.get(&2).unwrap().value());
 //! ```
 //!
+//! # Features
+//!
+//! If compiled with the `rayon` feature, some parallel traits will be implemented for
+//! [`ConMap`] and [`ConSet`]. Currently, [`FromParallelIterator`] and [`ParallelExtend`]
+//! are supported.
```

Comment:
Suggestion for slight future proofing: we're getting some more types in addition to `Conmap` and `ConSet` soonish. So maybe say `some parallel traits will be implemented for the types from this crate`?